.github/workflows/renovate-changesets.yml

```
@@ -0,0 +1,127 @@
+            function getChangedChangesetFiles() {
+              const changedFiles = execCommand('git diff --name-only HEAD~1 HEAD').split('\n');
+              return changedFiles.filter(file => 
+                file.startsWith('.changeset/') && 
+                file.endsWith('.md') && 
+                !file.includes('README.md')
+              );
+            }
+            
+            function isChangesetCreatedByBot(changesetFiles) {
+              if (changesetFiles.length === 0) return false;
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)